docs/current/guides/snippets/use-secrets/external/main.py

```diff
@@ -25,14 +25,14 @@ async def main():
     # print result
     print(out)
 
-def gcp_get_secret_plaintext(project_id, secret_id):
-    secret_uri = f"projects/{project_id}/secrets/{secret_id}/versions/1"
+async def gcp_get_secret_plaintext(project_id, secret_id):
+    secret_uri = f"projects/{project_id}/secrets/{secret_id}/versions/latest"
 
     # initialize Google Cloud API client
-    client = secretmanager.SecretManagerServiceClient()
+    client = secretmanager.SecretManagerServiceAsyncClient()
 
     # retrieve secret
-    response = client.access_secret_version(request={"name": secret_uri})
+    response = await client.access_secret_version(request={"name": secret_uri})
```

Comment:
@vikram-dagger I didn't run the pipeline myself, but if you confirm that this is working as expected, then 👍